### PR TITLE
Added show_code feature

### PR DESCRIPTION
**Pull request by johnson-52197**
#15 

If show_code is set to True, it will create a new cell and paste the code that was used to generate the answer.

### CODE REVIEW

**Comment by gventuri** [comment]
Some tests are failing: could you check it out?

Also, maybe we should find a way to make this feature only available if it's in a notebook.
Check it out: https://stackoverflow.com/questions/15411967/how-can-i-check-if-code-is-executed-in-the-ipython-notebook

What do you think?

**Comment by johnson-52197** [comment]
Sure @gventuri I'll check the failing cases. 

Thank you for the resource, I'll include the ipython notebook checks as well.

**Comment by johnson-52197** [comment]
All the unit test cases have passed successfully when running the application locally.

**Comment by johnson-52197** [comment]
Yet to include the notebook check.

**Comment by gventuri** [comment]
@johnson-52197 I'd say in that case we could `self.log()`, so it will only print it `if self._verbose == True:`. What do you think?

**Comment by johnson-52197** [comment]
That makes more sense @gventuri. I'll modify it.

**Comment by johnson-52197** [comment]
@gventuri included all the changes. Please review it.

File: pandasai/__init__.py [hunk 1]
```diff
@@ -172,6 +193,11 @@ def run_code(
             return result
         except:
             return captured_output
+        
+    def create_new_cell(self, contents: str):
```
Comment (gventuri) [inline pandasai/__init__.py:197]: Maybe we can move it to a dedicate helper (like notebook or anything like that?

In this way we keep the code cleaner.

File: pandasai/__init__.py [hunk 2]
```diff
@@ -59,6 +62,18 @@ def __init__(
         self._is_conversational_answer = conversational
         self._verbose = verbose
         self._enforce_privacy = enforce_privacy
+        self._is_notebook = self.in_notebook()
+
+    def in_notebook(self):
```
Comment (gventuri) [inline pandasai/__init__.py:67]: Maybe we can move it to a dedicate helper (like notebook or anything like that?

In this way we keep the code cleaner.

**Review by gventuri — CHANGES_REQUESTED** [review]
Almost there @johnson-52197!

**Comment by johnson-52197** [comment]
Sure @gventuri. 

I could think of two approaches.
1. Creating a folder named helpers inside pandasai and include notebook.py (we can add more helpers in the future)
2. a single python file named notebook.py inside pandasai.

Which one would you prefer?

File: pandasai/__init__.py (hunk 2 above)
Comment (johnson-52197) [reply pandasai/__init__.py:67]: > Sure @gventuri.
> 
> I could think of two approaches.
> 
> 1. Creating a folder named helpers inside pandasai and include notebook.py (we can add more helpers in the future)
> 2. a single python file named notebook.py inside pandasai.
> 
> Which one would you prefer?

**Comment by gventuri** [comment]
Great job @johnson-52197! I like the 1 better! It scales very well, let's go for it! 🚀

**Comment by johnson-52197** [comment]
@gventuri done. Please review.

**Review by gventuri — APPROVED** [review]

**Pull request merged**